pages/chapter2.tsx

```diff
@@ -0,0 +1,41 @@
+import PageCard from '@/components/PageCard';
```

Comment:
No tests for this file?

Reply:
Change is added but, the test was giving me errors as chapter2.tsx starts in lowercase and the inside was declared in lowercase, I had to change it to capital, is that correct? If I change the file to capitalcase, the page is missing

Reply:
You can import it like this

```js
import Chapter2 from './chapter2';

```